projects.html

```diff
@@ -109,7 +108,60 @@
                     {% endfor %}
                 </div>
             </section>
-            <!-- End of Projects Section -->
```

Comment:
it is end of projects secion

add a comment end of projects section 

and add another comment start of student projects 

they should be seperate